woocommerce/wc-admin
# Remove product settings video note

Fixes #5083

This removes the "Learn more about Product Settings" note, both from new stores (by deleting the note factory class) and from existing stores (by adding it as an obsolete note).

### Detailed test instructions:

- On a store with the "Learn more about Product Settings" note present, bump up the version in `FeaturePlugin.php` to trigger an install and delete the note
- Verify that the note has been removed
- Delete all of the existing notes and run the `wc_admin_daily` cron task
- Verify that the note has not been recreated

### Changelog Note:

Dev: Remove product settings video note
